src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.incidents.models import IncidentActivity, IncidentActivityType, IncidentStatus
+from sentry.models import Project
+
+
+class TeamAlertsTriggeredEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a time-bucketed (by day) count of triggered alerts owned by a given team.
+        """
+        project_list = Project.objects.get_for_team_ids([team.id])
+        owner_ids = [team.actor_id] + [om.user.actor_id for om in team.member_set]
+        start, end = get_date_range_from_params(request.GET)
+        bucketed_alert_counts = list(
+            IncidentActivity.objects.filter(
+                type__in=[
+                    IncidentActivityType.CREATED.value,
+                    IncidentActivityType.STATUS_CHANGE.value,
+                ],
+                value__in=[
+                    None,
+                    IncidentStatus.OPEN,
+                    IncidentStatus.CRITICAL,
+                    IncidentStatus.WARNING,
+                ],
+                incident__alert_rule__owner__in=owner_ids,
+                incident__projects__in=project_list,
```

Comment:
I think this join might cause us to duplicate incidents with multiple projects. Could you confirm what the query looks like?

Reply:
Discussed on Slack, using this query:
```
SELECT DATE_TRUNC('day', "sentry_incidentactivity"."date_added" AT TIME ZONE 'UTC') AS "bucket",
       COUNT("sentry_incidentactivity"."id") AS "count"
FROM "sentry_incidentactivity"
INNER JOIN "sentry_incident" ON ("sentry_incidentactivity"."incident_id" = "sentry_incident"."id")
INNER JOIN "sentry_alertrule" ON ("sentry_incident"."alert_rule_id" = "sentry_alertrule"."id")
WHERE (("sentry_incidentactivity"."type" = 1
        OR ("sentry_incidentactivity"."type" = 2
            AND "sentry_incidentactivity"."value" IN (NONE,
                                                      IncidentStatus.OPEN,
                                                      IncidentStatus.CRITICAL,
                                                      IncidentStatus.WARNING)))
       AND "sentry_incidentactivity"."date_added" >= 2021-06-30 20:20:04.019365+00:00
       AND "sentry_incidentactivity"."date_added" <= 2021-09-28 20:20:04.019365+00:00
       AND "sentry_alertrule"."owner_id" IN (4,
                                             2)
       AND "sentry_incidentactivity"."incident_id" IN
         (SELECT V0."incident_id"
          FROM "sentry_incidentproject" V0
          WHERE V0."project_id" IN
              (SELECT U0."id"
               FROM "sentry_project" U0
               INNER JOIN "sentry_projectteam" U1 ON (U0."id" = U1."project_id")
               WHERE (U0."status" = 0
                      AND U1."team_id" IN (3)))))
GROUP BY DATE_TRUNC('day', "sentry_incidentactivity"."date_added" AT TIME ZONE 'UTC')
```